build.sbt

```diff
@@ -595,6 +595,21 @@ def isInTestShard(name: String, logger: Logger): Boolean = {
   }
 }
 
+lazy val metalsDependencies = project
```

Comment:
I will still keep the dependencies in a separate project, since intransitive prints warnings for each dependency.